integration-tests/tests/src/tests/sync/app.ts

```diff
@@ -133,6 +133,44 @@ describe("App", () => {
     });
   });
 
+  describe("how to handle metadata", () => {
+    afterEach(async () => {
+      Realm.clearTestState();
+    });
+
+    it("is accessible", () => {
+      expect(MetadataMode).deep.equals({
+        NoEncryption: "noEncryption",
+        Encryption: "encryption",
+        NoMetadata: "noMetadata",
+      });
+    });
+
+    it("persists a user but does not encrypt it", () => {
+      const config: AppConfiguration = { id: "smurf", metadata: { mode: MetadataMode.NoEncryption } };
```

Comment:
I wonder why this was not exposed as `metadataMode` instead of an object with a single `mode` property. @kneth do we reasonably expect this to contain more properties in the future?